python/spacewalk/server/importlib/headerSource.py

```diff
@@ -644,7 +645,8 @@ def createPackage(
 
     # bug #524231 - we need to call fullFilelist() for RPM v3 file list
     # to expand correctly
-    header.hdr.fullFilelist()
+    if not ignore_fullFileList:
+        header.hdr.fullFilelist()
```

Comment:
Double negation can be hard to read, can we change this to 
```
if expand_full_filelist:
    header.hdr.fullFileList()
```

Reply:
I agree..done